mill.hpp

```diff
@@ -95,7 +97,7 @@ class Cutter: public RoutingMill {
  */
 /******************************************************************************/
 class Driller: public Mill
-{
+{ 
```

Comment:
No need for this change, right?

If at all, I'd have put `{ }` on the end of line 99.